ToolBox/QuantConnect.ToolBox.csproj

```diff
@@ -155,6 +155,8 @@
   <ItemGroup>
     <Compile Include="GDAXDownloader\GDAXDownloader.cs" />
     <Compile Include="GDAXDownloader\Program.cs" />
+    <Compile Include="GzipStreamProvider.cs" />
+    <Compile Include="IvolatilityEquityConverter\Program.cs" />
```

Comment:
It looks like the actual file on disk uses `IVolatilityEquityConverter` (capital V) -- while this will go unnoticed on windows, on any case sensitive file system (such as used in unix systems) it will cause an error.

Reply:
Sorry Im actually working in Rider on Mac, so this was running fine and wen't unnoticed. Im guessing VS support for those files is much better.